microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -431,6 +437,226 @@ func (r *GMConnectorReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	return ctrl.Result{}, nil
 }
 
+// read the configmap file from the manifests
+// update the values of the fields in the configmap
+// add service details to the fields
+func preProcessUserConfigmap(ctx context.Context, dynamicClient *dynamic.DynamicClient, ns string, hwType string, gmcGraph *mcv1alpha3.GMConnector) error {
+	var cfgFile string
+	// var adjustFile string
+	if hwType == xeon {
+		cfgFile = yaml_dir + "/qna_configmap_xeon.yaml"
```

Comment:
not sure why path has qna ?  But we can deal with that later.

Reply:
it's defined in the manifest